libs/failsafe-controller/src/main/java/net/jodah/failsafe/internal/ControlledScheduler.java

```diff
@@ -1,122 +0,0 @@
-/**
```

Comment:
ℹ️ The logic here is somewhat moved into the new ThreadMonitor class and replaced with a simple anonymous class created by `ThreadMonitor.monitor(Scheduler)`.